MLS.Agent.Tests/ApiViaHttpTestsBase.cs

```diff
@@ -63,7 +63,7 @@ protected ApiViaHttpTestsBase(ITestOutputHelper output)
             StartupOptions options = null)
         {
             HttpResponseMessage response;
-            using (var agent = new AgentService(options ?? StartupOptions.FromCommandLine("hosted")))
+            using (var agent = new AgentService(options ?? StartupOptionsFactory.CreateFromCommandLine("hosted")))
```

Comment:
Why introduce a factor class for this? I think this makes it less discoverable.

Reply:
this FromCommandLine functionality is only used in tests. What would be a better name for that?

Reply:
I see. Personally, I'd still leave it in the product code so people stumble onto it more easily.